packages/ui/src/ui/pages/system/Masters/Instance.tsx

```diff
@@ -88,10 +102,12 @@ export const Instance: FC<Props> = ({instance, hostType, allowVoting, allowServi
                     {allowService && (
                         <ChangeMaintenanceButton
                             className={b('host-btn')}
-                            address={address}
+                            path={instance.getPath()}
+                            title={`Edit ${shortName}`}
```

Comment:
`shortName` doesn't look like a good choice here. It is better to show full address (not path).